plugins/scale/wayfire/plugins/scale-transform.hpp

```diff
@@ -232,6 +243,21 @@ class scale_transformer_t : public wf::view_2D
      */
     wlr_box get_bounding_box(wf::geometry_t view, wlr_box region) override
     {
+        if (view != last_view_box)
```

Comment:
Why is this needed? If another transformer has changed, it should call damage() (before rendering). When it calls damage(), this method will be called, so I don't think you need an idle call and special handling here (even if the pre hooks indicate a size change).

Reply:
I see, so there is no need to call damage here, just update our padding